axon-framework/queries/query-dispatchers.md

```diff
@@ -227,5 +227,132 @@ commandGateway.sendAndWait(new RedeemCmd("gc1", amount));
 
    Since we subscribed to updates with the `println()` method, the update will be printed out once it is received.
 
+
+### Streaming queries
+
+The Streaming query allows a client to stream large database result sets. The Streaming query relies on reactive stream model, specifically the `Flux` type. Using streaming query, does not imply that reactive db drivers need to be used, users can hook to the stream and control the flow of data.
+
+
+Streaming query is flexible enough to be used with **any** query return type. That means that any return type that is not Flux will automatically be converted to Flux, based on that Flux will return single or multiple items.
+
+Query gateway provides nifty `streamingQuery` method to utilize streaming query easily. 
+It's simple to use, and requires just two parameters: query and expected response type class.
+`streamingQuery` method **is lazy**, meaning query is sent once Flux is subscribed to.
+
+
+It is also possible to utilize streaming query with `query` method: `queryGateway.query(query,  ResponseTypes.fluxOf(CardSummary.class))`.
+However, this approach is **eager** and less intuitive as it requires more boilerplate code.
+In this case `query` method will send query immediately and return CompletableFuture which returns a payload of type Flux, that needs to be subscribed to.
```

Comment:
To further clarify what you mean with "the query method," linking to that section in the documentation is helpful:

```suggestion
It is also possible to utilize the streaming query with the  [Point-to-Point query](query-dispatchers.md#point-to-point-queries) method: `queryGateway.query(query,  ResponseTypes.fluxOf(CardSummary.class))`.
However, this approach is **eager** and less intuitive as it requires more boilerplate code.
In this case, the `query` method will send the query immediately and return a `CompletableFuture`.
This `CompletableFuture` returns a payload of type `Flux` that you subscribe to.
```

Reply:
this is outdated, and will be removed, but will commit suggestion for now